Action.podspec

```
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Action"
-  s.version      = "3.10.0"
+  s.version      = "3.10.2"
```

Shouldn't this be `3.10.1` ? 
```suggestion
  s.version      = "3.10.1"
```

---

Nop.. I checked and `3.10.1` is already there. it was make for Carthage releases.. So this is why I choose the next in order to do a release that covers both Carthage and Cocoapods